data/anaconda.conf

```diff
@@ -174,6 +174,64 @@ default_scheme = LVM
 luks_version = luks2
 
 
+[Storage Constraints]
+
+# Minimal size of the total memory.
+min_ram = 320 MiB
+
+# Minimal size of the available memory for LUKS2.
+luks2_min_ram = 128 MiB
+
+# Should we recommend to specify a swap partition?
+swap_is_recommended = True
+
+# Recommended minimal sizes of partitions.
+# Specify a mount point and a size on each line.
+min_partition_sizes =
+    /      250 MiB
+    /usr   250 MiB
+    /tmp   50  MiB
+    /var   384 MiB
+    /home  100 MiB
+    /boot  200 MiB
+
+# Required minimal sizes of partitions.
+# Specify a mount point and a size on each line.
+req_partition_sizes =
+
+# Allowed device types of the / partition if any.
+# Valid values:
+#
+#  LVM        Allow LVM.
+#  MD         Allow RAID.
+#  PARTITION  Allow standard partitions.
+#  BTRFS      Allow Btrfs.
+#  DISK       Allow disks.
+#  LVM_THINP  Allow LVM Thin Provisioning.
+#
+root_device_types =
+
+# Mount points that must be on a linux file system.
+# Specify a list of mount points.
+must_be_on_linuxfs = / /var /tmp /usr /home /usr/share /usr/lib
+
+# Mount points that must be on the / file system.
+# Specify a list of mount points.
+must_be_on_root = /bin /dev /sbin /etc /lib /root /mnt lost+found /proc
+
+# Mount points that must NOT be on the / file system.
+# Specify a list of mount points.
+must_not_be_on_root =
+
+# Mount points that are recommended to be reformatted.
+#
+# It will be recommended to create a new file system on a mount point
+# that has a whitelisted prefix, but doesn't have a blacklisted one.
+# Specify lists of mount points.
+reformat_whitelist = /boot /var /tmp /usr
+reformat_blacklist = /home /usr/local /opt /var/www
```

Comment:
What if something is not on whitelist neither blacklist? What then will happen?
Also why `/` is not specified here?

Reply:
> What if something is not on whitelist neither blacklist? What then will happen?

We will not show a message with the recommendation. Is it really so surprising?

> Also why `/` is not specified here?

We require to reformat the file system on `/`. It is a different check.

Reply:
OK, I guess this should be enough.